File: Clientside/Admin/src/models/VariantOrderDto.ts
import OrderDto from "./OrderDto";
import SizeDto from "./SizeDto";
import VariantDto from "./VariantDto";

interface VariantOrderDto {
    id: number
    sizeId: number
    size: SizeDto
    variantId: number
    variant: VariantDto
    orderId: number
    orderedItemsTotal: number
    price: number
    discountPrice?: number
}

export default VariantOrderDto

File: Serverside/Admin/Api/Dtos/Order/VariantOrderDto.cs
namespace FNO.Admin.AdminApi.Dtos.Order
{
    public class VariantOrderDto
    {
        public int Id { get; set; }
        public int SizeId { get; set; }
        public SizeDto Size { get; set; }
        public int VariantId { get; set; }
        public GetVariantFromOrderDto Variant { get; set; }
        public int OrderId { get; set; }
        public int OrderedItemsTotal { get; set; }
        public decimal Price { get; set; }
        public decimal? DiscountPrice { get; set; }
    }
}
